src/Expression.ts

```diff
@@ -32,7 +32,11 @@
  */
 import * as M from 'fp-ts/lib/Monoid'
 import * as T from 'fp-ts/lib/Traced'
-import { identity, Endomorphism, flow } from 'fp-ts/lib/function'
+import * as R from 'fp-ts/lib/Record'
+import * as S from 'fp-ts/lib/string'
+import * as B from 'fp-ts/lib/boolean'
+import { identity, Endomorphism, flow, pipe } from 'fp-ts/lib/function'
+import { Lens } from 'monocle-ts'
```

Comment:
We should probably be using the experimental `Lens` implementation from `monocle-ts` instead of the class-based implementation, since the latter will likely be deprecated when `fp-ts@3.0.0` is released in favor of the experimental modules.

So the import should be `import * as L from 'monocle-ts/Lens'`.

Reply:
Done